.github/workflows/dev_builds.yml

```
@@ -0,0 +1,39 @@
+name: Dev builds
+
+on:
+  push:
+    branches:
+      - main
+      - 'release-v[0-9]+.[0-9]+'
```

I think we shouldn't restrict job to minor versions

```suggestion
      - 'release-v[0-9]+'
      - 'release-v[0-9]+.[0-9]+'
```